test/util/vault/util.go

```diff
@@ -63,12 +64,17 @@ func NewVaultInitializer(container, rootMount, intermediateMount, role string) (
 
 	client.SetToken(vaultToken)
 
+	if authPath == "" {
+		authPath = "approle"
+	}
```

Comment:
We're defaulting this field in multiple places - I'd rather return an error if it isn't specified than default to the same value again